src/include/uapi/ipc/control.h

```diff
@@ -152,4 +152,19 @@ struct sof_ipc_ctrl_data {
 	};
 } __attribute__((packed));
 
+/* event type */
+enum sof_ipc_ctrl_event_type {
+	SOF_CTRL_EVENT_KD_NOTIFICATION = 0, /**< keyword detection event */
+};
+
+struct sof_ipc_comp_event {
+	struct sof_ipc_reply rhdr;
+	uint32_t src_comp_id; /* source component id */
+	int type;	/* event type - SOF_CTRL_EVENT_* */
```

Comment:
Please be consistent with the indentation of comments. It looks like the preferred way is to use a tab after ; then add the comment.

Reply:
files updated